packages-ts/gauntlet-terra-contracts/src/lib/inspection.ts

```diff
@@ -12,3 +19,23 @@ export const getLatestOCRConfigEvent = async (provider: LCDClient, contract: Acc
 
   return setConfigTx?.logs?.[0].eventsByType['wasm-set_config']
 }
+
+export const getLatestOCRNewTransmissionEvent = async (provider: LCDClient, contract: AccAddress) => {
+  let transmissionTx = (
+    await provider.tx.search({
+      events: [
+        {
+          key: 'wasm-new_transmission.contract_address',
+          value: contract,
+        },
+      ],
+      'pagination.limit': '1',
+      order_by: 'ORDER_BY_DESC',
+    })
+  ).txs
+
+  return transmissionTx.length > 0 ? transmissionTx[0]?.logs?.[0].eventsByType['wasm-new_transmission'] : null
+}
+
+export const parseObserversByLength = (observers: string, observersNumber: number): number[] =>
```

Comment:
Bring this function closer where it is used. If in the future is useful in somewhere else, we could bring it to a common place then